docs/core/deploying/native-aot/index.md

```diff
@@ -114,24 +122,48 @@ By default, native AOT publishing produces debug information in a separate file
 
 ## Limitations of native AOT deployment
 
-Native AOT applications come with a few fundamental limitations and compatibility issues. The key limitations include:
+### [.NET 7](#tab/net7)
+
+Native AOT apps have limitations and compatibility issues:
 
-- No dynamic loading (for example, `Assembly.LoadFile`).
-- No run-time code generation (for example, `System.Reflection.Emit`).
+- No dynamic loading, for example, `Assembly.LoadFile`.
+- No run-time code generation, for example, `System.Reflection.Emit`.
 - No C++/CLI.
-- No built-in COM (only applies to Windows).
+- Windows: No built-in COM.
 - Requires trimming, which has [limitations](../trimming/incompatibilities.md).
 - Implies compilation into a single file, which has known [incompatibilities](../single-file/overview.md#api-incompatibility).
 - Apps include required runtime libraries (just like [self-contained apps](../index.md#publish-self-contained), increasing their size as compared to framework-dependent apps).
 - <xref:System.Linq.Expressions> always use their interpreted form, which is slower than run-time generated compiled code.
+- Should be targeted for console type apps, ASP.NET Core is not supported.
+- Not all the runtime libraries are fully annotated to be native AOT compatible. That is, some warnings in the runtime libraries aren't actionable by end developers.
+- Limited diagnostic support for debugging and profiling.
 
-The publish process analyzes the entire project and its dependencies and produces warnings whenever the limitations of the published application might be encountered at run time.
+The publish process:
 
-The first release of native AOT in .NET 7 has some more limitations:
+- Analyzes the entire project and its dependencies.
+- Produces warnings whenever the limitations of the published app might be encountered at run time.
+
+### [.NET 8+](#tab/net8plus)
 
-- Should be targeted for console type applications (not ASP.NET Core).
-- Not all the runtime libraries are fully annotated to be native AOT compatible (that is, some warnings in the runtime libraries aren't actionable by end developers).
-- Limited diagnostic support (debugging and profiling).
+Native AOT apps have limitations and compatibility issues:
```

Comment:
In the .NET 8 list I am wondering if it would make sense to explicitly call out that we support ASP.NET minimal APIs with native AOT with a few limitations?

Reply:
> In the .NET 8 list I am wondering if it would make sense to explicitly call out that we support ASP.NET minimal APIs with native AOT with a few limitations?

Good suggestion, I'll add that and link off.

Reply:
However, not all of ASP.NET is supported. Just minimal APIs and gRPC. MVC, Razor Pages, Blazor, and SignalR are all unsupported.

Reply:
We should just link to https://learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?view=aspnetcore-8.0